packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1751,4 +1740,51 @@ async function revokeCertifiedAttribute(
   } satisfies Tenant;
 }
 
+function validateAddress(address: string): string {
```

Comment:
IMHO there are too many steps in order to validate a mail, are these really necessary?
With the regex we don't need the extra steps of removing characters and spaces since it's already included.

I suggest to use this one: `^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$` (taken from [here](https://www.regular-expressions.info/email.html))? **Note: the regex must be case insensitive for it to work properly.**

There's no perfect regex for validating a mail, but this one is simpler and covers most of the cases.

Reply:
I would prefer to leave `emailPattern` as it is so that it is aligned with the frontend as well, since it is the same pattern they use